index.html

```diff
@@ -602,298 +609,143 @@ <h3>Ethereum EIP712 Signature 2021</h3>
-        edentialSchema":[{"name":"id","type":"string"},{"name":"type","type":"string"}],"Credentia
-        lSubject":[{"name":"type","type":"string"},{"name":"id","type":"string"},{"name":"name","t
-        ype":"string"},{"name":"child","type":"Person"}],"EIP712Domain":[{"name":"name","type":"st
-        ring"},{"name":"version","type":"string"},{"name":"chainId","type":"uint256"},{"name":"sal
-        t","type":"bytes32"}],"Person":[{"name":"type","type":"string"},{"name":"name","type":"str
-        ing"}],"Proof":[{"name":"verificationMethod","type":"string"},{"name":"created","type":"st
-        ring"},{"name":"proofPurpose","type":"string"},{"name":"type","type":"string"}],"Verifiabl
-        eCredential":[{"name":"@context","type":"string[]"},{"name":"type","type":"string[]"},{"na
-        me":"id","type":"string"},{"name":"issuer","type":"string"},{"name":"issuanceDate","type":
-        "string"},{"name":"credentialSubject","type":"CredentialSubject"},{"name":"credentialSchem
-        a","type":"CredentialSchema"},{"name":"proof","type":"Proof"}]}}
-      </pre>
+      <section>
+        <h3>Basic Document - Types Generation - No Embedding</h3>
 
-    <p>The following is the resulting <code>proof</code> object:</p>
-    <pre class="example">
-        {
-          "type": "EthereumEip712Signature2021",
-          "created": "2021-07-09T19:47:41Z",
-          "proofPurpose": "assertionMethod",
-          "proofValue": "0x5fb8f18f21f54c2df8a2720d0afcee7dbbb18e4b7a22ce6e8183633d63b076d329122584db769cd78b6cd5a7094ede5ceaa43317907539187f1f0d8875f99e051b",
-          "verificationMethod": "did:example:aaaabbbb#issuerKey-1",
-          "eip712": {
+        <p>
+          With the following <code>inputOptions</code> provided to the signature suite along with the
+          <code>testBasicDocument</code> input document:
+        </p>
+
+        <pre class="example">
+          {
+            "date": "2021-08-30T13:28:02Z",
+            "verificationMethod": "did:pkh:eip155:1:0xAED7EA8035eEc47E657B34eF5D020c7005487443#blockchainAccountId",
             "domain": {
-              "chainId": 4,
-              "name": "https://example.com",
-              "salt": "0x000000000000000000000000000000000000000000000000aaaabbbbccccdddd",
-              "version": "2"
-            },
+              "name": "Test"
+            }
+          }
+        </pre>
+
+        <p>The following is the resulting <code>proof</code> object:</p>
+
+        <pre class="example">
+          {
+            "created": "2021-08-30T13:28:02Z",
+            "proofPurpose": "assertionMethod",
+            "proofValue": "0xbbdf2914c7572185bbc263e066dfb43f3136e4441fddb3fe3ea4541bbf7fd1f00d8e5af3ce4fbb1f2ebd5256f39b22cef7f285189df2976ea0c385c77f0a42791b",
+            "type": "EthereumEip712Signature2021",
+            "verificationMethod": "did:pkh:eip155:1:0xAED7EA8035eEc47E657B34eF5D020c7005487443#blockchainAccountId",
+          }
+        </pre>
```

Comment:
I haven't been able to verify this test vector ("Basic Document - Types Generation - No Embedding" in [28bd5ed](https://github.com/w3c-ccg/ethereum-eip712-signature-2021-spec/commit/28bd5edecde8395242aea8ba64e9be25f59585d0)). I tried various workarounds.
I understand this test vector is supposed to demonstrate auto-generation of types and the info that would otherwise be in the `eip712` proof property which is omitted here. But I think it may be problematic that the domain parameter `"name": 'test"` is passed in as an option, but does not appear in the resulting proof/document. I assume that verifiable credentials are generally supposed to be self-contained, and DIDs are typically publicly-resolvable, such that everything needed to verify the proof/signature is in the proof/document or in the resolved DID document / verification method object, or specified in the signature suite. If it is really intended to be able to issue verifiable credentials / create proof objects with this suite that require an external domain property to be passed in during issuance and verification, perhaps this a note/warning could be added about this. Otherwise, maybe the `"domain": "test"` here could be removed, or changed to a well-known/public value specified in this spec; or it could be passed through in the proof object (e.g. at `proof.eip712.domain` like in the other examples). In any case, I could use some additional info/help to be able to verify this test vector, e.g. the whole TypedData message used for signing/verification, or the parameters passed to the sign/verify function.

Reply:
Opened issue #42 about this. Adding a note linking to it in 15c959335ff1417ea374d7bb3917e8f7b91c16fe (#43).